feat(beahviors): Allow encoders to use any behavior (#1275)
zmkfirmware/zmk: docs/docs/behaviors/sensor-rotate.md

---
title: Sensor Rotation
sidebar_label: Sensor Rotation
---

## Summary

> The Sensor Rotation behavior sends a different keypress, depending on whether the sensor is rotated clockwise or counter-clockwise.

Reviewer: Maybe change "sends a different keypress" into "triggers a configured behavior"

Author: I think this wording makes more sense and I've added it to the pr, thanks for the suggestion!

> - If rotated counter-clockwise, the first keycode is sent.

Reviewer: Instead of "keycode is sent" maybe "behavior is triggered"

- If rotated clockwise, the second keycode is sent.

### Configuration

> An example of how to implement an encoder to change rgb brightness:

Reviewer 2: Another minor detail in passing: suggesting capitalization of "RGB" for consistency across the docs.
```suggestion
An example implementation of an encoder that changes RGB brightness is shown below:
```

> 

Reviewer 3: We need an example for the -var and non-var variants here.

```
/ {
    behaviors {
        rgb_encoder: rgb_encoder {

>             compatible = "zmk,behavior-mod-morph";

Reviewer 2: `zmk,behavior-mod-morph` should be replaced with `zmk,behavior-sensor-rotate`
